api/resourcedeleter.go

```diff
@@ -1,102 +1,97 @@
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
-	if res, err := deleteDeployment(namespace, deployName, k8sClient); res != "" {
-		result = append(result, res)
-		if err != nil {
-			return result, err
-		}
+	res, err = deleteDeployment(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
-	if err := deleteRedisFailover(namespace, deployName, k8sClient); err != nil {
+	res, err = deleteRedisFailover(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
 		result = append(result, err.Error())
 	}
 
-	if res, err := deleteSecret(namespace, deployName, k8sClient); res != "" {
-		result = append(result, res)
-		if err != nil {
-			return result, err
-		}
+	res, err = deleteSecret(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
-	if res, err := deleteIngress(namespace, deployName, k8sClient); res != "" {
-		result = append(result, res)
-		if err != nil {
-			return result, err
-		}
+	res, err = deleteIngress(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
-	if res, err := deleteAutoscaler(namespace, deployName, k8sClient); res != "" {
-		result = append(result, res)
-		if err != nil {
-			return result, err
-		}
+	res, err = deleteAutoscaler(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
-	if res, err := deleteConfigMapRules(namespace, deployName, k8sClient); res != "" {
-		result = append(result, res)
-		if err != nil {
-			return result, err
-		}
+	res, err = deleteConfigMapRules(namespace, deployName, k8sClient)
+	result = append(result, res)
+	if err != nil {
+		return result, err
 	}
 
 	return result, nil
 }
 
 func deleteService(namespace string, deployName string, k8sClient kubernetes.Interface) (result string, e error) {
 	if err := k8sClient.CoreV1().Services(namespace).Delete(deployName, &k8smeta.DeleteOptions{}); err != nil {
-		return filterNotFound(fmt.Sprintf("could not delete service: %s in namespace: %s: %s", deployName, namespace, err), err)
+		return filterNotFound(fmt.Sprintf("service: "), err)
 	}
-	return "", nil
+	return "service: OK", nil
 }
 
 func deleteDeployment(namespace string, deployName string, k8sClient kubernetes.Interface) (result string, e error) {
 	deploymentDeleteOption := k8smeta.DeletePropagationForeground
-	if err := k8sClient.ExtensionsV1beta1().Deployments(namespace).Delete(deployName, &k8smeta.DeleteOptions{ PropagationPolicy: &deploymentDeleteOption, }); err != nil {
-		return filterNotFound(fmt.Sprintf("could not delete deployment: %s in namespace: %s: %s", deployName, namespace, err), err)
+	if err := k8sClient.ExtensionsV1beta1().Deployments(namespace).Delete(deployName, &k8smeta.DeleteOptions{PropagationPolicy: &deploymentDeleteOption}); err != nil {
+		return filterNotFound("deployment: ", err)
```

Comment:
skulle kanskje stått `deployment: failed` eller `deployment: NOT OK`? Liker `failed` best.

Reply:
eller `FAIL` som jeg ser du har brukt lengre ned.

Reply:
N/A eller FAIL legges på av filterNotFound